pelita/scripts/pelita_main.py

```diff
@@ -224,16 +233,18 @@ def long_help(s):
                            help='Initialize the random number generator with SEED.')
 game_settings.add_argument('--allow-camping', const=True, action='store_const',
                            help='Food does not age when in a bot’s shadow')
+game_settings.add_argument('--food', type=str, metavar="T:F", default=None,
+                        help="Distribute F food pellets for each team, with T being the number "
+                        "of food pellets that are \"trapped\" within tunnels and chambers. "
+                        " If not set use maze size specific defaults.")
 
 layout_opt = game_settings.add_mutually_exclusive_group()
 layout_opt.add_argument('--layout', metavar='LAYOUT',
-                        help='Use maze layout specified in LAYOUT. LAYOUT can be'
-                             ' a file containing a valid layout or the name of a '
-                             'built-in layout. Use --list-layouts to get a list '
-                             'of all available layouts.')
-layout_opt.add_argument('--size', metavar='STRING', default='normal',
+                        help='Use maze layout specified in LAYOUT. LAYOUT is'
+                             ' a file containing a valid layout string.')
```

Comment:
I think we can simplify this to `'Use layout from FILE'` and have `'FILE'` (or FILENAME) as the metavar. I think it is obvious that the layout file contains something like a layout string.

Reply:
fixed in 3efd0dcc